msmbuilder/featurizer/featurizer.py

```diff
@@ -347,6 +348,99 @@ def partial_transform(self, traj):
         return np.hstack(x)
 
 
+
+class AlphaAngleFeaturizer(Featurizer):
+    """Featurizer to extract alpha (dihedral) angles.
+
+    The alpha angle of residue `i` is the dihedral formed by the four CA atoms
+    of residues `i-1`, `i`, `i+1` and `i+2`.
+
+    Parameters
+    ----------
+    sincos : bool
+        Instead of outputting the angle, return the sine and cosine of the
+        angle as separate features.
+    """
+
+    def __init__(self, sincos=True):
+        self.sincos = sincos
+        self.atom_indices = None
+
+    def partial_transform(self, traj):
+
```

Comment:
Make the change, commit, push to the same branch `brookee:alphas`, and it will update here